/*
 * MiniStocks created by Daniel Kostuj
 *
 * This file contains all declarations for the Ticker refresh thread.
 *
 * Use of this source code is governed by the license that can be
 * found in the LICENSE file.
 */

#ifndef TICKERREFRESHTHREAD_H
#define TICKERREFRESHTHREAD_H

#include "ui_mainwindow.h"
#include "../dialogs/MainWindow.h"

void refreshTicker(Ui::MainWindow *ui);


#endif // TICKERREFRESHTHREAD_H
